tools/expotools/src/commands/UpdateVersionsEndpoint.ts

```diff
@@ -42,7 +56,61 @@ function setSdkVersionConfig(sdkVersionConfig: object, key: string, value: any):
   }
 }
 
+async function applyChangesToStagingAsync(delta: any, previousVersions: any, newVersions: any) {
+  if (!delta) {
+    console.log(chalk.yellow('There are no changes to apply in the configuration.'));
+    return;
+  }
+
+  console.log(`\nHere is the diff of changes to apply on ${chalk.green('staging')} version config:`);
+  console.log(
+    jsondiffpatch.formatters.console.format(delta!, previousVersions),
+  );
+
+  const isCorrect = await askForCorrectnessAsync();
+
+  if (isCorrect) {
+    // Save new configuration.
+    try {
+      await Versions.setVersionsAsync(newVersions);
+    } catch (error) {
+      console.error(error);
+    }
+
+    console.log(
+      chalk.green('\nSuccessfully updated staging config. You can check it out on'),
+      chalk.blue(`https://${STAGING_HOST}/--/api/v2/versions`),
+    );
+  } else {
+    console.log(chalk.yellow('Canceled'));
+  }
+}
+
+async function revertStagingConfigurationAsync() {
+  // Get current production config.
+  Config.api.host = PRODUCTION_HOST;
+  const productionVersions = await Versions.versionsAsync();
+
+  // Wait for the cache to invalidate.
+  await sleepAsync(1000);
```

Comment:
lol

Reply:
🤷‍♂ actually we don't need to sleep for one second, in `promote-versions` command we wait just 10ms, so I'll change that